user/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path
+
+from user import views
+
+urlpatterns = [
+    path('', views.BookView.as_view(), name="form"),
```

Comment:
make 'users' as url blank url used to show homepage or index page.

Reply:
UserView name =user, user